dask/bytes/core.py

```diff
@@ -476,6 +476,20 @@ def get_fs(protocol, storage_options=None):
                         "    pip install gcsfs")
         cls = _filesystems[protocol]
 
+    elif protocol in ['adl', 'adlfs']:
+
+        import_required('dask_adlfs',
+                        "Need to install `dask_adlfs` and `azure-storage-py"
+                        "thon` libraries for Azure Datalake Storage support.\n"
```

Comment:
The clipping of `thon` here is a bit awkward.  Same below with the `pip install` line.  

Also, can we push a release onto PyPI?